pkg/delete/delete.go

```diff
@@ -151,7 +150,7 @@ func deleteSettingsObject(ctx context.Context, c dtclient.Client, entries []Dele
 			log.WithCtxFields(ctx).Debug("Deleting settings object %s/%s with objectId %s.", e.Type, e.Identifier, obj.ObjectId)
```

Comment:
Project is missing here (it's optional, but can be present). I'd suggest including it using the coordinate.

Additionally, in the error below, we should also include it.

Reply:
Addressed by introducing a String() method on DeletePointer that will cleanly log in colon separated format with and without project being present